content/en/continuous_integration/tests/ruby.md

```diff
@@ -502,11 +264,6 @@ All other [Datadog Tracer configuration][6] options can also be used.
 
 {{< partial name="whats-next/whats-next.html" >}}
 
-
-[1]: /agent/
-[2]: https://docs.datadoghq.com/agent/cluster_agent/admission_controller/
-[3]: https://app.datadoghq.com/organization-settings/api-keys
-[4]: /tracing/trace_collection/dd_libraries/ruby/#installation
```

Comment:
Link 4 is still needed, it's referenced on line 72

Reply:
@hestonhoffman fixed the link, thanks!